.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
```

Comment:
@RobinCK Can you remove this file please.

We already use a linter and we would prefer to add a lint check on the master branch on github as a build.